spec/rails_helper.rb

```diff
@@ -56,13 +56,22 @@
       Capybara::Selenium::Driver.new(app, browser: :chrome)
     end
     Capybara.javascript_driver = :selenium_chrome
+
+    Capybara::Screenshot.register_driver(:selenium_chrome) do |driver, path|
+      driver.browser.save_screenshot(path)
+    end
   else
     Capybara.register_driver :selenium_firefox do |app|
       Capybara::Selenium::Driver.new(app, browser: :firefox)
     end
     Capybara.javascript_driver = :selenium_firefox
+
+    Capybara::Screenshot.register_driver(:selenium_firefox) do |driver, path|
+      driver.browser.save_screenshot(path)
+    end
   end
 
+  Capybara.default_wait_time = 15
```

Comment:
@samnang this was recommended by travis-ci for failed tests with capybara element not found.
under their common build errors documents. However, this was not why the test were failing. That was later solved by the update dependencies branch. Some updated npm modules had broken the app and the builds during fresh installs.